javascript-light-bulb/main.js

```diff
@@ -0,0 +1,15 @@
+var clicksTotal = 0;
+var lightOff = document.querySelector('.light-on');
+
+lightOff.addEventListener('click', function () {
+  clicksTotal++;
+  if (clicksTotal % 2 !== 0) {
```

Comment:
Interesting way to handle the switching on and off!

One thought to simplify, since you don't really need to know the number of times clicked, `clicksTotal` can set to a `boolean` instead of a `number`

Then in your handler function, you can set the variable storing the boolean to be opposite of what the current value is.... like this:

```
var isOn = false;

isOn = !isOn;
```